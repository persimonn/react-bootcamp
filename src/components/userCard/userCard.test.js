import React from "react";
import toJson from "enzyme-to-json";
import { shallow } from "enzyme";
import UserCard, { sum } from "./index.js";

describe("testing postCard", () => {
  const wrapper = shallow(<UserCard />);

  it("renders properly", () => {
    expect(wrapper.length).toEqual(1);
  });

  it("renders snapshot", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
