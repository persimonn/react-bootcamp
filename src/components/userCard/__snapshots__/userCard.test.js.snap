// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing postCard renders snapshot 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <div
      className="col"
    >
      <img
        style={
          Object {
            "width": "60px",
          }
        }
      />
    </div>
    <div
      className="col"
    >
      <a
        className="userNameLink"
        href="#"
      />
    </div>
  </div>
</div>
`;
